EdFi.Buzz.UI/src/Feature/Privacy/index.tsx

```diff
@@ -32,137 +34,50 @@ const BackToHome = styled(Link)`
-          Users may have had with the Website. To unsubscribe to such announcements, Users should follow the “unsubscribe”
-          instructions provided in the email sent, or contact&nbsp;
-        <a href="mailto:communications@ed-fi.org" target="_blank" rel="noopener noreferrer">communications@ed-fi.org</a>.</p>
-        <h3>User Assumption of Risk; Release and Indemnification</h3>
-        <p>Users assume all risks related to use of the Website. Each User agrees to indemnify, defend,
-          and hold EFA (including its owners, heirs, successors, assigns, affiliates
-          (including but not limited to the Michael &amp; Susan Dell Foundation), officers, directors, members,
-          employees, agents and representatives) harmless from and against any losses, damages, liabilities or costs of
-          any nature related in any manner to the use of the Website by such User. EFA shall not be liable to Users or
-          any other parties for any claims for damages, whether direct, indirect, special, consequential, incidental
-          or punitive, based on any cause of action whatsoever relating to use of the Website by any person or entity.</p>
-        <h3>Users Who are Governmental Employees of Officials</h3>
-        <p>EFA welcomes governmental employees and officials to peruse the EFA Website.
-          If you choose to contact EFA for further information, you acknowledge that you are contacting
-          EFA on behalf of your governmental body/agency to request the input and expertise of EFA for technical
-          assistance on a specific project.</p>
-        <h3>Governing Law</h3>
-        <p>
-          These Terms of Use are governed by the laws of the State of Texas, United States of America,
-          without regard to choice of law provisions.
-          The courts located in Austin, Texas, USA, shall have exclusive jurisdiction over any disputes relating to the Website.</p>
-        <h3>Amendment</h3>
-        <p>These Terms of Use may be amended at any time by EFA in its sole discretion.
-          These Terms of Use constitute the entire agreement between EFA and the User relating to use of the Website.
-        </p>
-        <h3>Acceptance of Terms of Use</h3>
-        <p>YOUR USE AND VIEWING OF THE WEBSITE INDICATES YOUR AFFIRMATIVE AGREEMENT TO THESE TERMS OF USE.
-          IF YOU DO NOT AGREE WITH THESE TERMS OF USE, PLEASE DO NOT USE THE WEBSITE.</p>
-      </section>
-    </article>
-    <BackToHome
-      to={{
-        pathname: '/'
-      }}>
-      <LeftArrowIcon />
-      <div>Go back to Home</div>
-    </BackToHome>
-  </MainContainer>
-);
+const Privacy: React.FunctionComponent<PrivacyProps> = ({title}) => {
+  const [markdownFile,setMarkdownFile] = useState(null);
+
+  useEffect(() => {
+    let isMounted = true; // note this flag denote mount status
+    const readmePath = (`http://${window.location.host}/PRIVACY.POLICY.md`);
```

Comment:
http protocol will cause issues if the user does not enable origin that with CORS. Can we omit that?

Reply:
Updated to use window.location.origin instead of the hardcoded http.